Sources/SotoCore/Tracing/Baggage+ServiceName.swift

```diff
@@ -0,0 +1,19 @@
+import Baggage
+import Tracing
+
+public enum SotoServiceBaggageKey: Baggage.Key {
+    public typealias Value = SotoServiceBaggage
+}
+
+public struct SotoServiceBaggage {
+    public let serviceName: String
+    public let databaseSystem: String?
```

Comment:
`databaseSystem` seems very specific to certain services. Should this not go into `serviceAttributes`

Reply:
Please see my comment below: https://github.com/soto-project/soto-core/pull/443#issuecomment-850554367